Library/Formula/libmxml.rb

```diff
@@ -20,6 +20,10 @@ def install
                           "--enable-shared",
                           "--prefix=#{prefix}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/mxmldoc", "--version"
```

Comment:
Would be great to try and write out a source/header file and generate documentation from it: http://www.msweet.org/documentation/project3/Mini-XML.html#MXMLDOC